packages/ckeditor5-utils/src/dom/emittermixin.js

```diff
@@ -179,6 +179,7 @@ extend( ProxyEmitter.prototype, EmitterMixin, {
 	 * @param {Object} [options={}] Additional options.
 	 * @param {Boolean} [options.useCapture=false] Indicates that events of this type will be dispatched to the registered
 	 * listener before being dispatched to any EventTarget beneath it in the DOM tree.
+	 * @param {Boolean} [options.usePassive=false] Indicates that the function specified by listener will never call preventDefault().
```

Comment:
Well, this is only half of the story. The real use-case for this option is unblocking the browser's main thread and this should be mentioned.

Reply:
Updated.